bot/errors.py

```diff
@@ -1,6 +1,9 @@
 from __future__ import annotations
 
-from typing import Hashable, TYPE_CHECKING
+from typing import Hashable, TYPE_CHECKING, Union
```

Comment:
Doesn't `TYPE_CHECKING` come first?

Reply:
I don't really understand what you're saying here.

The Pydis style guide states that for imports:
> Each group must be ordered alphabetically, with uppercase modules coming before lowercase.
and that's what I'm doing here? H < T < U so `Hashable, TYPE_CHECKING, Union`, as I did.